src/Umbraco.Web.UI.Client/src/views/propertyeditors/blockgrid/prevalue/umbBlockGridAreaEditor.component.js

```diff
@@ -34,7 +64,7 @@
             }
         });
 
-    function BlockGridAreaController($scope, $element, assetsService, localizationService, editorService, overlayService) {
+    function BlockGridAreaController($scope, localizationService, editorService, overlayService) {
```

Comment:
It looks like the binding of "model" is a complex object. Do we need the double-binding `=` in that case or would a reference now work `<`?

Reply:
Oh, yes. And because we are not in the Content-Editor this complex model does not update. I have changed it.